src/app/store/effects/auth.effects.ts

```diff
@@ -22,7 +22,7 @@ import { IAppState } from 'src/app/store/state/app.state';
 @Injectable()
 export class AuthEffects {
   private readonly BACKEND_ERRORS = {
-    Unauthorized: 'user.auth.sign-in.account-has-been-deleted',
+    Unauthorized: 'user.auth.sign-in.bad-email-or-password',
```

Comment:
Please remove this change since it does not fix the issue because this message was hardcoded in the effect. I fixed this problem in #3269